CHANGELOG.md

```diff
@@ -15,9 +15,11 @@
 
 ### `ConstantReader`
 
-* Added `isAny`.
-* Allow `anyValue` to return `null` if the value is actually `null`.
+* Added `isAny` (deprecated) and `isLiteral`.
```

Comment:
Maybe make a note about why we're adding isAny for consistency even though it's immediately deprecated.